### ISSUE TITLE
could you add  coordinate on board and label numbers on stones

### ISSUE THREAD

**Issue by l1t1** [fsparv/LeelaWatcher#2]
help for understand very long time games

**Comment by fsparv** [comment]
I agree this would be nice, but it's going to take me getting motivated to dig into the graphics drawing code. I haven't done custom component graphics in over a decade so that's a pretty big hurdle. Patches welcome of course :).

**Comment by l1t1** [comment]
could you show me which part of code draw the stones?

**Comment by fsparv** [comment]
Board painting his here: 

https://github.com/fsparv/LeelaWatcher/blob/master/src/main/java/leelawatcher/gui/ImageMaker.java#L133

**Comment by l1t1** [comment]
thanks. I am not good at gui java, only found
for (int lines = 1; lines < size + 1; lines++) { 
171       x1 = lineSp * lines; 
172       x2 = lineSp * lines; 
173       y1 = lineSp; 
174       y2 = makeSize - lineSp; 
175       aLine.setLine(x1, y1, x2, y2); 
176       BGraphs.setColor(Color.black); 
177       BGraphs.draw(aLine); 
178     } 
could be
y1 = lineSp; 
y2 = makeSize - lineSp; 
BGraphs.setColor(Color.black); 

for (int lines = 1; lines < size + 1; lines++) { 
  x1 = lineSp * lines; 
  aLine.setLine(x1, y1, x1, y2); 
  BGraphs.draw(aLine); 
}

**Comment by fsparv** [comment]
I appreciate all suggestions, but if you want to suggest a code change, the best way to communicate clearly is to fork the repository, make the change, test to see that it has the desired effect and then issue a pull request (referencing an issue hopefully). I do this for free, and for fun, and trying to eyeball-diff unformatted code half of it with line numbers half without is not really "fun" :).

**Comment by l1t1** [comment]
like http://www.yss-aya.com/cgos/viewer.cgi?19x19/SGF/2017/12/07/324556.sgf

**Comment by fsparv** [comment]
I'm fully aware of what board coordinates might look like, having played go online since 1996, I've seen it many times (see my ancient and outdated site: www.gameofgo.info for the story of how I got started). If/when I have time I'll add them. If you want the feature faster than that, you can contribute a patch. https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project and other similar links you'll find by googling "contribute to github project" detail the normal process for contributions to open source projects on github.
